boskos/boskos.go

```diff
@@ -34,13 +34,16 @@ import (
 )
 
 const (
-	defaultSyncPeriod = 10 * time.Minute
+	defaultSyncPeriod      = 10 * time.Minute
+	defaultRequestTTL      = 30 * time.Second
+	defaultRequestGCPeriod = time.Minute
 )
 
 var (
 	configPath        = flag.String("config", "config.yaml", "Path to init resource file")
 	storagePath       = flag.String("storage", "", "Path to persistent volume to load the state")
 	syncPeriod        = flag.Duration("sync-period", defaultSyncPeriod, "Period at which to sync config")
+	requestTTL        = flag.Duration("request-ttl", defaultRequestTTL, "request TTL before being removed from Priority queue")
```

Comment:
Technically you're just added to the end? Not removed, right?

Reply:
Yes you are removed if you don't send updates, but if you do send another update after ttl expired you will be added back to the end. I will rename to "losing your priority"